packages/frontend/src/features/semanticViewer/runners/SemanticViewerResultsRunner.ts

```diff
@@ -231,17 +203,14 @@ export class SemanticViewerResultsRunner extends ResultsRunner {
 
         // The backend call has no knowledge of field types, so we need to map them to the correct types
         const vizColumns: VizColumn[] =
-            SemanticViewerResultsRunner.convertColumnsToVizColumns(
+            SemanticViewerResultsRunner.convertColumnNamesToVizColumns(
```

Comment:
make this the backend's problem

Reply:
Agree. I'd like to do it in a follow up. I will add a comment.

Reply:
Resolving but leaving the other question about this open